src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private User petro;
+    private User semen;
+    private User vasyl;
+    private User dmytro;
+    private User kotyhoroshko;
+    private User ivanMykhailovych;
+    private List<User> userList;
+
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeEach
+    void setUp() {
+        petro = new User("Petro", "Pet", 34);
+        semen = new User("Semen", "SeMen2005", 16);
+        vasyl = new User("Vasyl", null, 41);
```

Comment:
```suggestion
        nullPassUser = new User("Vasyl", null, 41);
```
для читаймості краще буте так називати вхідні дані